src/components/atoms/bar-chart-content.tsx

```diff
@@ -54,16 +54,18 @@ export const BarChartContent: FC<BarChartContentProps> = ({
     return (
       <G>
         {data.map((value, index) => (
-          <Rect
-            x={x(index)}
-            y={y(value) - cornerRoundness}
-            rx={cornerRoundness}
-            ry={cornerRoundness}
-            width={bandwidth}
-            height={cornerRoundness * 2} // Height of the Rect
-            fill={secondaryColor}
-            key={`bar-${index}`}
-          />
+          <>
```

Comment:
Don't need this leftover fragment, there's only one child (and it's probably what's causing the missing key warning)